src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -20,6 +20,17 @@
     <Compile Include="$(DeviceRoot)**/*.cs" Exclude="@(_ExcludeFromCompile)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="MathHelper.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(TargetFramework) == 'netstandard2.0'">
```

Comment:
I think you will need to exclude CreditCard stuff for netstandard2.0 build as well which I'm not sure if it's ok